.github/workflows/integration.yml

```diff
@@ -274,12 +286,29 @@ jobs:
           CLIENT_SECRET: ${{ secrets.EDGEDNS_CLIENT_SECRET }}
         run: |
           touch values.yaml
+          [[ '${{ inputs.dns }}' == 'az_dns' ]] && echo "$AZ_DNS" >> values.yaml && domainSuffix="--set cluster.domainSuffix=tst-${{ github.run_id }}.aks.redkubes.net"
```

Comment:
Are these needed? I think we do not use azure anymore right?

Reply:
Good find, I think this because a merge conflict.